### ISSUE TITLE
[Feature request] Add max output for yaw, roll, x, y and z

### ISSUE THREAD

**Issue by bababoombatman** [opentrack/opentrack#1241]
In the mapping menu we can lower the max output for pitch to 90°, but this option is not present in the yaw, roll, x, y and z menu. Would it be possible to add them? It would help with precision mapping when higher values are not needed.

Unrelated, but i would also like to ask if it's possible to add even lower values to pitch max output setting. For example 45°.

Thanks in advance

**Comment by sthalik** [comment]
No, almost everyone needs full motion range. There's no need to clutter the UI.

**Comment by diVineProportion** [comment]
@bababoombatman you know you can do this right? 
![image](https://user-images.githubusercontent.com/38984875/112568142-3dbc5000-8d9f-11eb-8bdb-d6932da1df61.png)
